packages/e2e/test/wallet_epoch_3/PersonalWallet/conwayTransactions.test.ts

```diff
@@ -226,34 +206,59 @@ describe('PersonalWallet/conwayTransactions', () => {
     assertTxHasCertificate(confirmedTx, certificate);
   };
 
-  const isRegisteredDRep = async () => {
-    await waitForWalletStateSettle(dRepWallet);
-    return await firstValueFrom(dRepWallet.governance.isRegisteredAsDRep$);
-  };
+  beforeEach(async () => {
+    const faucet = (await getWallet({ env, logger, name: 'Faucet Wallet', polling: { interval: 50 } })).wallet;
+    await walletReady(faucet);
+
+    const mnemonics = util.generateMnemonicWords();
```

Comment:
In the optic to run e2e tests on real networks as well, this makes us to loose ADA or tADA.

To follow the directive to not use faucet wallets for tests, but funding test dedicated wallets, wouldn't be better to use the provided mnemonics and using faucet wallet account (`idx: 0`) to fund test dedicated accounts (`idx: n` where `n > 0`)?